### ISSUE TITLE
Clustering example

### ISSUE THREAD

**Issue by edublancas** [ploomber/ploomber#359]
*Note: working on this requires some familiarity with ploomber* 

We have a [classification example](https://github.com/ploomber/projects/tree/master/ml-basic) but we're missing a clustering one. It'd be to have one that has the following tasks:

1. loading data (can be some [sample dataset](https://scikit-learn.org/stable/modules/classes.html#module-sklearn.datasets))
2. cleaning data (sample datasets are in good shape but adding some cleaning steps is good for the sake of the example)
3. visualize clean data
4. train model
5. evaluate model

The structure is:

```
load -> clean -> viz
clean -> train -> evaluate
```
Please make each task a script.

Whatever metric/plot you use to evaluate the model, please add a short explanation and a link to learn more.

Check out the [contributing guide](https://github.com/ploomber/projects/blob/master/CONTRIBUTING.md) for details.

**Comment by edublancas** [comment]
wrong repo. moved to: https://github.com/ploomber/projects/issues/12
